src/main/scala/io/iohk/ethereum/vm/EvmConfig.scala

```diff
@@ -0,0 +1,177 @@
+    case object G_balance extends GasCost
+    case object G_sload extends GasCost
+    case object G_jumpdest extends GasCost
+    case object G_sset extends GasCost
+    case object G_sreset extends GasCost
+    case object R_sclear extends GasCost
+    case object R_selfdestruct extends GasCost
+    case object G_selfdestruct extends GasCost
+    case object G_create extends GasCost
+    case object G_codedeposit extends GasCost
+    case object G_call extends GasCost
+    case object G_callvalue extends GasCost
+    case object G_callstipend extends GasCost
+    case object G_newaccount extends GasCost
+    case object G_exp extends GasCost
+    case object G_expbyte extends GasCost
+    case object G_memory extends GasCost
+    case object G_txcreate extends GasCost
+    case object G_txdatazero extends GasCost
+    case object G_txdatanonzero extends GasCost
+    case object G_transaction extends GasCost
+    case object G_log extends GasCost
+    case object G_logdata extends GasCost
+    case object G_logtopic extends GasCost
+    case object G_sha3 extends GasCost
+    case object G_sha3word extends GasCost
+    case object G_copy extends GasCost
+    case object G_blockhash extends GasCost
+    case object G_extcodesize extends GasCost
+    case object G_extcodecopy_base extends GasCost
+    case object G_selfdestruct_to_new_account extends GasCost
+  }
+
+  val FrontierFeeSchedule: FeeSchedule = FeeSchedule(Map(
+      GasCost.G_zero -> UInt256(0),
+      GasCost.G_base -> UInt256(2),
+      GasCost.G_verylow -> UInt256(3),
+      GasCost.G_low -> UInt256(5),
+      GasCost.G_mid -> UInt256(8),
+      GasCost.G_high -> UInt256(10),
+      GasCost.G_extcode -> UInt256(700),
+      GasCost.G_balance -> UInt256(20),
+      GasCost.G_sload -> UInt256(50),
+      GasCost.G_jumpdest -> UInt256(1),
+      GasCost.G_sset -> UInt256(20000),
+      GasCost.G_sreset -> UInt256(5000),
+      GasCost.R_sclear -> UInt256(15000),
+      GasCost.R_selfdestruct -> UInt256(24000),
+      GasCost.G_selfdestruct -> UInt256(0),
+      GasCost.G_create -> UInt256(32000),
+      GasCost.G_codedeposit -> UInt256(200),
+      GasCost.G_call -> UInt256(40),
+      GasCost.G_callvalue -> UInt256(9000),
+      GasCost.G_callstipend -> UInt256(2300),
+      GasCost.G_newaccount -> UInt256(25000),
+      GasCost.G_exp -> UInt256(10),
+      GasCost.G_expbyte -> UInt256(10),
+      GasCost.G_memory -> UInt256(3),
+      GasCost.G_txcreate -> UInt256(21000),
+      GasCost.G_txdatazero -> UInt256(4),
+      GasCost.G_txdatanonzero -> UInt256(68),
+      GasCost.G_transaction -> UInt256(21000),
+      GasCost.G_log -> UInt256(375),
+      GasCost.G_logdata -> UInt256(8),
+      GasCost.G_logtopic -> UInt256(375),
+      GasCost.G_sha3 -> UInt256(30),
+      GasCost.G_sha3word -> UInt256(6),
+      GasCost.G_copy -> UInt256(3),
+      GasCost.G_blockhash -> UInt256(20),
+      GasCost.G_extcodesize -> UInt256(20),
+      GasCost.G_extcodecopy_base -> UInt256(20),
```

Comment:
What's this? How is it different from `G_extcode` (which doesn't appear to be used anymore)?